integrations/salesforce/subscription.go

```diff
@@ -105,6 +105,17 @@ func (h handler) eventLoop(ctx context.Context, client pb.PubSubClient, subscrib
 				l.Error("ChangeEventHeader is not a map in event data")
 				continue
 			}
+
+			// TODO(INT-329): Temporary filter to ignore self-triggered events.
+			changeOrigin, ok := header["changeOrigin"].(string)
+			if !ok {
+				l.Error("changeOrigin is not a string in ChangeEventHeader")
```

Comment:
Since this log message is specific to Salesforce, mention Salesforce somehwhere in this message, so we know what it means when we see it in Datadog